apps/task.go

```diff
@@ -44,7 +44,7 @@ type TasksResponse struct {
 
 func FindTaskByID(id TaskID, tasks []Task) (Task, bool) {
 	for _, task := range tasks {
-		if task.ID == id {
+		if strings.HasPrefix(task.ID.String(), id.String()) && len(id.String) > 36 {
```

Comment:
id.String -> id.String()

Reply:
I already saw it from the failing build, it was a last minute fix (and not doing the test locally).... 😬